apps/system/js/app_window.js

```diff
@@ -265,6 +260,11 @@
    */
   AppWindow.prototype.setVisible =
     function aw_setVisible(visible) {
+      if (this._visible == visible) {
```

Comment:
nit, s/==/===/ (I guess? I suppose if you want to support calling it with non-boolean values, maybe == is better?)